### ISSUE TITLE
Column Ordering: name, role and state indications are missing

### ISSUE THREAD

**Issue by steve-haar** [vmware-clarity/starters#82]
From Amy Li,

> No name, role, and state indicating to assistive technologies that these are draggable items

**Comment by AmyLiNow** [comment]
@steve-haar this is related to an invalid heading controls issue (https://github.com/vmware-clarity/starters/issues/83)
adding more invalid code into the datagrid headings will add to that issue. It is recommended to provide separate controls (outside of the data grid) for reordering columns using assistive technology in addition to mouse drag and drop.
